birju

```diff
@@ -0,0 +1,2 @@
+I have added few lines in birju file.
+And the problem is resolved.

```

Comment:
I dont want this line.
Please remove it.

Reply:
ok.

Reply:
check now.

Reply:
All right, thanks.